carma-messenger-core/Dockerfile

```diff
@@ -11,13 +11,15 @@
 #  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #  License for the specific language governing permissions and limitations under
 #  the License.
-
-FROM usdotfhwastol/carma-base:carma-system-4.5.0 as base_image
+ARG ORGANIZATION="usdotfhwastoldev"
+ARG TAG="develop"
+FROM ${ORGANIZATION}/carma-base:${TAG} as base_image
 FROM base_image as setup
+ARG BRANCH="develop" 
```

Comment:
BRANCH variable can be more explicit as it can be confused with the branch of carma-messenger itself to someone not paying attention. Something like, DEPENDENCY_CHECKOUT_BRANCH would be better.

Reply:
Addressed